### ISSUE TITLE
Stupid Icon

### ISSUE THREAD

**Issue by MoshDev** [oarrabi/Guaka#60]
Can you change this stupid Icon?

**Comment by oarrabi** [comment]
Yes, this is something we are working on. We will keep this for now and go for a different one later
